detox/src/android/espressoapi/ViewActions.js

```diff
@@ -40,6 +65,24 @@ class ViewActions {
     };
   }
 
+  static click(rollbackAction) {
+    if (!rollbackAction) {
```

Comment:
@DanielMSchmidt , I see unit test failing on this (`src/android/expect.js`). Is this a reason why you hesitated to add `isThruthy` check?

Reply:
I will take another look at the unit tests 👍 No, I hesitated because I don't see so much value 🤷‍♂️

Reply:
I should have hesitated because of this. The problem is it is optional and not required and we have no way to check if an argument is required or not. So all checks we do should test if the thing that is passed is what it promises to be if something is passed. If null is passed we won't be able to guard us on this side of the fence. I will do a quick research if there is any hypothetical way for us to know if sth is optional and I will let you know.